cli/cmd/profile.go

```diff
@@ -35,30 +50,51 @@ func newCmdProfile() *cobra.Command {
 	options := newProfileOptions()
 
 	cmd := &cobra.Command{
-		Use:   "profile [flags] --template (SERVICE)",
+		Use:   "profile [flags] (--template | --open-api file) (SERVICE)",
 		Short: "Output template service profile config for Kubernetes",
 		Long: `Output template service profile config for Kubernetes.
 		
-		This outputs a service profile template for the given service.  Edit the
-		template and then apply it with kubectl to add a service profile to a
-		service.
+		This outputs a service profile for the given service.
```

Comment:
Not directly related to your change, but can you un-indent this entire string block? Otherwise the help text is weirdly indented when printed by the CLI:

```
Output template service profile config for Kubernetes.
		
		This outputs a service profile for the given service.
		
		If the --template flag is specified, it outputs a service profile template.
...
```

See some of the other files in this directory for an example of how they handle the long description.